Facebook.Android/build.cake

```diff
@@ -1,7 +1,7 @@
 #addin nuget:?package=Cake.FileHelpers&version=3.2.1
 
-var FB_VERSION = "11.2.0";
-var NUGET_VERSION = "11.2.0.1";
+var FB_VERSION = "13.1.0";
+var NUGET_VERSION = "13.1.0";
```

Comment:
Do you see additional warnings when building your updated version?

Same question, different way to ask: Are we sure that no modifications to metadata.xml files are needed?

Reply:
Actually yes there are quite a few warnings :( 
```
1>CLASSPARSE : warning : no match for AccessTokenSource.valueOf (rex: <span class="sympad"><a href=".*com/facebook/AccessTokenSource.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for FacebookRequestError.Category.valueOf (rex: <span class="sympad"><a href=".*com/facebook/FacebookRequestError.Category.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for FacebookSdk.getApplicationSignature (rex: <span class="sympad"><a href=".*com/facebook/FacebookSdk.html#getApplicationSignature\(android.content.Context\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for FacebookSdk.publishInstallAsync (rex: <span class="sympad"><a href=".*com/facebook/FacebookSdk.html#publishInstallAsync\(android.content.Context, java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for FacebookSdk.setDataProcessingOptions (rex: <span class="sympad"><a href=".*com/facebook/FacebookSdk.html#setDataProcessingOptions\(java.lang.String\[\]\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for FacebookSdk.setDataProcessingOptions (rex: <span class="sympad"><a href=".*com/facebook/FacebookSdk.html#setDataProcessingOptions\(java.lang.String\[\], int, int\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestAsyncTask.doInBackground (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestAsyncTask.html#doInBackground\(java.lang.Object\[\]\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestAsyncTask.onPostExecute (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestAsyncTask.html#onPostExecute\(java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.add (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#add\(int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.add (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#add\(java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.contains (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#contains\(com.facebook.GraphRequest\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.contains (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#contains\(java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.get (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#get\(int\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.indexOf (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#indexOf\(com.facebook.GraphRequest\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.indexOf (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#indexOf\(java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.lastIndexOf (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#lastIndexOf\(com.facebook.GraphRequest\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.lastIndexOf (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#lastIndexOf\(java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.remove (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#remove\(int\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.remove (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#remove\(int\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.remove (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#remove\(com.facebook.GraphRequest\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.remove (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#remove\(java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphRequestBatch.set (rex: <span class="sympad"><a href=".*com/facebook/GraphRequestBatch.html#set\(int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GraphResponse.PagingDirection.valueOf (rex: <span class="sympad"><a href=".*com/facebook/GraphResponse.PagingDirection.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for HttpMethod.valueOf (rex: <span class="sympad"><a href=".*com/facebook/HttpMethod.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for LoggingBehavior.valueOf (rex: <span class="sympad"><a href=".*com/facebook/LoggingBehavior.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AppEventsLogger.FlushBehavior.valueOf (rex: <span class="sympad"><a href=".*com/facebook/appevents/AppEventsLogger.FlushBehavior.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AppEventsLogger.ProductAvailability.valueOf (rex: <span class="sympad"><a href=".*com/facebook/appevents/AppEventsLogger.ProductAvailability.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AppEventsLogger.ProductCondition.valueOf (rex: <span class="sympad"><a href=".*com/facebook/appevents/AppEventsLogger.ProductCondition.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for FlushResult.valueOf (rex: <span class="sympad"><a href=".*com/facebook/appevents/FlushResult.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for PerformanceGuardian.UseCase.valueOf (rex: <span class="sympad"><a href=".*com/facebook/appevents/PerformanceGuardian.UseCase.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for UserDataStore.access$getExternalHashedUserData$p (rex: <span class="sympad"><a href=".*com/facebook/appevents/UserDataStore.html#access$getExternalHashedUserData$p\(com.facebook.appevents.UserDataStore\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for UserDataStore.access$getInitialized$p (rex: <span class="sympad"><a href=".*com/facebook/appevents/UserDataStore.html#access$getInitialized$p\(com.facebook.appevents.UserDataStore\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for UserDataStore.access$getInternalHashedUserData$p (rex: <span class="sympad"><a href=".*com/facebook/appevents/UserDataStore.html#access$getInternalHashedUserData$p\(com.facebook.appevents.UserDataStore\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for UserDataStore.access$getSharedPreferences$p (rex: <span class="sympad"><a href=".*com/facebook/appevents/UserDataStore.html#access$getSharedPreferences$p\(com.facebook.appevents.UserDataStore\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for UserDataStore.access$getTAG$p (rex: <span class="sympad"><a href=".*com/facebook/appevents/UserDataStore.html#access$getTAG$p\(com.facebook.appevents.UserDataStore\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for UserDataStore.access$initAndWait (rex: <span class="sympad"><a href=".*com/facebook/appevents/UserDataStore.html#access$initAndWait\(com.facebook.appevents.UserDataStore\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for UserDataStore.access$setSharedPreferences$p (rex: <span class="sympad"><a href=".*com/facebook/appevents/UserDataStore.html#access$setSharedPreferences$p\(com.facebook.appevents.UserDataStore, android.content.SharedPreferences\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for UserDataStore.access$updateHashUserData (rex: <span class="sympad"><a href=".*com/facebook/appevents/UserDataStore.html#access$updateHashUserData\(com.facebook.appevents.UserDataStore, android.os.Bundle\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for UserDataStore.access$writeDataIntoCache (rex: <span class="sympad"><a href=".*com/facebook/appevents/UserDataStore.html#access$writeDataIntoCache\(com.facebook.appevents.UserDataStore, java.lang.String, java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for MetadataIndexer.onActivityResumed (rex: <span class="sympad"><a href=".*com/facebook/appevents/aam/MetadataIndexer.html#onActivityResumed\(android.app.Activity\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for MetadataRule.updateRules (rex: <span class="sympad"><a href=".*com/facebook/appevents/aam/MetadataRule.html#updateRules\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for CodelessLoggingEventListener.getOnClickListener (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/CodelessLoggingEventListener.html#getOnClickListener\(com.facebook.appevents.codeless.internal.EventBinding, android.view.View, android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for CodelessLoggingEventListener.getOnItemClickListener (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/CodelessLoggingEventListener.html#getOnItemClickListener\(com.facebook.appevents.codeless.internal.EventBinding, android.view.View, android.widget.AdapterView\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for CodelessLoggingEventListener.updateParameters (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/CodelessLoggingEventListener.html#updateParameters\(android.os.Bundle\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for CodelessLoggingEventListener.AutoLoggingOnClickListener.onClick (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/CodelessLoggingEventListener.AutoLoggingOnClickListener.html#onClick\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for CodelessManager.onActivityDestroyed (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/CodelessManager.html#onActivityDestroyed\(android.app.Activity\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for CodelessManager.onActivityPaused (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/CodelessManager.html#onActivityPaused\(android.app.Activity\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for CodelessManager.onActivityResumed (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/CodelessManager.html#onActivityResumed\(android.app.Activity\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for RCTCodelessLoggingEventListener.getOnTouchListener (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/RCTCodelessLoggingEventListener.html#getOnTouchListener\(com.facebook.appevents.codeless.internal.EventBinding, android.view.View, android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewIndexer.buildAppIndexingRequest (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/ViewIndexer.html#buildAppIndexingRequest\(java.lang.String, com.facebook.AccessToken, java.lang.String, java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewIndexer.sendToServerUnity (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/ViewIndexer.html#sendToServerUnity\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewIndexer.sendToServerUnityInstance (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/ViewIndexer.html#sendToServerUnityInstance\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for SensitiveUserDataUtils.isSensitiveUserData (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/SensitiveUserDataUtils.html#isSensitiveUserData\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.findRCTRootView (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#findRCTRootView\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.getChildrenOfView (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#getChildrenOfView\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.getClassTypeBitmask (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#getClassTypeBitmask\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.getDictionaryOfView (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#getDictionaryOfView\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.getExistingOnClickListener (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#getExistingOnClickListener\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.getExistingOnTouchListener (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#getExistingOnTouchListener\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.getHintOfView (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#getHintOfView\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.getParentOfView (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#getParentOfView\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.getTextOfView (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#getTextOfView\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.getTouchReactView (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#getTouchReactView\(float\[\], android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.isRCTButton (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#isRCTButton\(android.view.View, android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.isRCTRootView (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#isRCTRootView\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.isRCTTextView (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#isRCTTextView\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.isRCTViewGroup (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#isRCTViewGroup\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.setOnClickListener (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#setOnClickListener\(android.view.View, android.view.View.OnClickListener\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.updateAppearanceOfView (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#updateAppearanceOfView\(android.view.View, org.json.JSONObject, float\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewHierarchy.updateBasicInfoOfView (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/internal/ViewHierarchy.html#updateBasicInfoOfView\(android.view.View, org.json.JSONObject\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for InAppPurchaseAutoLogger.startIapLogging (rex: <span class="sympad"><a href=".*com/facebook/appevents/iap/InAppPurchaseAutoLogger.html#startIapLogging\(android.content.Context\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for InAppPurchaseBillingClientWrapper.getOrCreateInstance (rex: <span class="sympad"><a href=".*com/facebook/appevents/iap/InAppPurchaseBillingClientWrapper.html#getOrCreateInstance\(android.content.Context\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for InAppPurchaseBillingClientWrapper.queryPurchase (rex: <span class="sympad"><a href=".*com/facebook/appevents/iap/InAppPurchaseBillingClientWrapper.html#queryPurchase\(java.lang.String, java.lang.Runnable\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for InAppPurchaseBillingClientWrapper.queryPurchaseHistory (rex: <span class="sympad"><a href=".*com/facebook/appevents/iap/InAppPurchaseBillingClientWrapper.html#queryPurchaseHistory\(java.lang.String, java.lang.Runnable\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for InAppPurchaseEventManager.hasFreeTrialPeirod (rex: <span class="sympad"><a href=".*com/facebook/appevents/iap/InAppPurchaseEventManager.html#hasFreeTrialPeirod\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for InAppPurchaseLoggerManager.filterPurchaseLogging (rex: <span class="sympad"><a href=".*com/facebook/appevents/iap/InAppPurchaseLoggerManager.html#filterPurchaseLogging\(java.util.Map<java.lang.String, org.json.JSONObject>, java.util.Map<java.lang.String, org.json.JSONObject>\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for InAppPurchaseSkuDetailsWrapper.getSkuDetailsParams (rex: <span class="sympad"><a href=".*com/facebook/appevents/iap/InAppPurchaseSkuDetailsWrapper.html#getSkuDetailsParams\(java.lang.String, java.util.List<java.lang.String>\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for InAppPurchaseUtils.getClass (rex: <span class="sympad"><a href=".*com/facebook/appevents/iap/InAppPurchaseUtils.html#getClass\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for InAppPurchaseUtils.getMethod (rex: <span class="sympad"><a href=".*com/facebook/appevents/iap/InAppPurchaseUtils.html#getMethod\(java.lang.Class<\?>, java.lang.String, java.lang.Class<\?>...\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for InAppPurchaseUtils.invokeMethod (rex: <span class="sympad"><a href=".*com/facebook/appevents/iap/InAppPurchaseUtils.html#invokeMethod\(java.lang.Class<\?>, java.lang.reflect.Method, java.lang.Object, java.lang.Object...\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AppEventsLoggerUtility.GraphAPIActivityType.valueOf (rex: <span class="sympad"><a href=".*com/facebook/appevents/internal/AppEventsLoggerUtility.GraphAPIActivityType.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for FileDownloadTask.doInBackground (rex: <span class="sympad"><a href=".*com/facebook/appevents/internal/FileDownloadTask.html#doInBackground\(java.lang.Object\[\]\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for FileDownloadTask.onPostExecute (rex: <span class="sympad"><a href=".*com/facebook/appevents/internal/FileDownloadTask.html#onPostExecute\(java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ModelManager.access$addModels (rex: <span class="sympad"><a href=".*com/facebook/appevents/ml/ModelManager.html#access$addModels\(com.facebook.appevents.ml.ModelManager, org.json.JSONObject\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ModelManager.access$enableMTML (rex: <span class="sympad"><a href=".*com/facebook/appevents/ml/ModelManager.html#access$enableMTML\(com.facebook.appevents.ml.ModelManager\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ModelManager.access$fetchModels (rex: <span class="sympad"><a href=".*com/facebook/appevents/ml/ModelManager.html#access$fetchModels\(com.facebook.appevents.ml.ModelManager\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ModelManager.access$isValidTimestamp (rex: <span class="sympad"><a href=".*com/facebook/appevents/ml/ModelManager.html#access$isValidTimestamp\(com.facebook.appevents.ml.ModelManager, long\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ModelManager.access$parseJsonArray (rex: <span class="sympad"><a href=".*com/facebook/appevents/ml/ModelManager.html#access$parseJsonArray\(com.facebook.appevents.ml.ModelManager, org.json.JSONArray\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ModelManager.Task.valueOf (rex: <span class="sympad"><a href=".*com/facebook/appevents/ml/ModelManager.Task.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for RemoteServiceWrapper.EventType.valueOf (rex: <span class="sympad"><a href=".*com/facebook/appevents/ondeviceprocessing/RemoteServiceWrapper.EventType.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for RemoteServiceWrapper.ServiceResult.valueOf (rex: <span class="sympad"><a href=".*com/facebook/appevents/ondeviceprocessing/RemoteServiceWrapper.ServiceResult.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for SuggestedEventsManager.populateEventsFromRawJsonString (rex: <span class="sympad"><a href=".*com/facebook/appevents/suggestedevents/SuggestedEventsManager.html#populateEventsFromRawJsonString\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for SuggestedEventsManager.trackActivity (rex: <span class="sympad"><a href=".*com/facebook/appevents/suggestedevents/SuggestedEventsManager.html#trackActivity\(android.app.Activity\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ViewOnClickListener.onClick (rex: <span class="sympad"><a href=".*com/facebook/appevents/suggestedevents/ViewOnClickListener.html#onClick\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for Task.continueWhile$default (rex: <span class="sympad"><a href=".*com/facebook/bolts/Task.html#continueWhile$default\(com.facebook.bolts.Task, java.util.concurrent.Callable, com.facebook.bolts.Continuation, java.util.concurrent.Executor, com.facebook.bolts.CancellationToken, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for BoltsMeasurementEventListener.access$open (rex: <span class="sympad"><a href=".*com/facebook/internal/BoltsMeasurementEventListener.html#access$open\(com.facebook.internal.BoltsMeasurementEventListener\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for BoltsMeasurementEventListener.access$setSingleton$cp (rex: <span class="sympad"><a href=".*com/facebook/internal/BoltsMeasurementEventListener.html#access$setSingleton$cp\(com.facebook.internal.BoltsMeasurementEventListener\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for BoltsMeasurementEventListener.getInstance (rex: <span class="sympad"><a href=".*com/facebook/internal/BoltsMeasurementEventListener.html#getInstance\(android.content.Context\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for CallbackManagerImpl.RequestCodeOffset.valueOf (rex: <span class="sympad"><a href=".*com/facebook/internal/CallbackManagerImpl.RequestCodeOffset.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for FeatureManager.Feature.valueOf (rex: <span class="sympad"><a href=".*com/facebook/internal/FeatureManager.Feature.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for FetchedAppGateKeepersManager.setRuntimeGateKeeper$default (rex: <span class="sympad"><a href=".*com/facebook/internal/FetchedAppGateKeepersManager.html#setRuntimeGateKeeper$default\(java.lang.String, com.facebook.internal.gatekeeper.GateKeeper, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for FileLruCache.get$default (rex: <span class="sympad"><a href=".*com/facebook/internal/FileLruCache.html#get$default\(com.facebook.internal.FileLruCache, java.lang.String, java.lang.String, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for FileLruCache.openPutStream$default (rex: <span class="sympad"><a href=".*com/facebook/internal/FileLruCache.html#openPutStream$default\(com.facebook.internal.FileLruCache, java.lang.String, java.lang.String, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for ImageRequest.Builder.copy$default (rex: <span class="sympad"><a href=".*com/facebook/internal/ImageRequest.Builder.html#copy$default\(com.facebook.internal.ImageRequest.Builder, android.content.Context, android.net.Uri, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for NativeProtocol.access$fetchAllAvailableProtocolVersionsForAppInfo (rex: <span class="sympad"><a href=".*com/facebook/internal/NativeProtocol.html#access$fetchAllAvailableProtocolVersionsForAppInfo\(com.facebook.internal.NativeProtocol, com.facebook.internal.NativeProtocol.NativeAppInfo\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for NativeProtocol.access$getFacebookAppInfoList$p (rex: <span class="sympad"><a href=".*com/facebook/internal/NativeProtocol.html#access$getFacebookAppInfoList$p\(com.facebook.internal.NativeProtocol\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for NativeProtocol.access$getProtocolVersionsAsyncUpdating$p (rex: <span class="sympad"><a href=".*com/facebook/internal/NativeProtocol.html#access$getProtocolVersionsAsyncUpdating$p\(com.facebook.internal.NativeProtocol\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for SmartLoginOption.valueOf (rex: <span class="sympad"><a href=".*com/facebook/internal/SmartLoginOption.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for WorkQueue.addActiveWorkItem$default (rex: <span class="sympad"><a href=".*com/facebook/internal/WorkQueue.html#addActiveWorkItem$default\(com.facebook.internal.WorkQueue, java.lang.Runnable, boolean, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GateKeeper.copy$default (rex: <span class="sympad"><a href=".*com/facebook/internal/gatekeeper/GateKeeper.html#copy$default\(com.facebook.internal.gatekeeper.GateKeeper, java.lang.String, boolean, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GateKeeperRuntimeCache.dumpGateKeepers$default (rex: <span class="sympad"><a href=".*com/facebook/internal/gatekeeper/GateKeeperRuntimeCache.html#dumpGateKeepers$default\(com.facebook.internal.gatekeeper.GateKeeperRuntimeCache, java.lang.String, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GateKeeperRuntimeCache.getGateKeeper$default (rex: <span class="sympad"><a href=".*com/facebook/internal/gatekeeper/GateKeeperRuntimeCache.html#getGateKeeper$default\(com.facebook.internal.gatekeeper.GateKeeperRuntimeCache, java.lang.String, java.lang.String, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GateKeeperRuntimeCache.getGateKeeperValue$default (rex: <span class="sympad"><a href=".*com/facebook/internal/gatekeeper/GateKeeperRuntimeCache.html#getGateKeeperValue$default\(com.facebook.internal.gatekeeper.GateKeeperRuntimeCache, java.lang.String, java.lang.String, boolean, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GateKeeperRuntimeCache.resetCache$default (rex: <span class="sympad"><a href=".*com/facebook/internal/gatekeeper/GateKeeperRuntimeCache.html#resetCache$default\(com.facebook.internal.gatekeeper.GateKeeperRuntimeCache, java.lang.String, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GateKeeperRuntimeCache.setGateKeeper$default (rex: <span class="sympad"><a href=".*com/facebook/internal/gatekeeper/GateKeeperRuntimeCache.html#setGateKeeper$default\(com.facebook.internal.gatekeeper.GateKeeperRuntimeCache, java.lang.String, com.facebook.internal.gatekeeper.GateKeeper, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GateKeeperRuntimeCache.setGateKeepers$default (rex: <span class="sympad"><a href=".*com/facebook/internal/gatekeeper/GateKeeperRuntimeCache.html#setGateKeepers$default\(com.facebook.internal.gatekeeper.GateKeeperRuntimeCache, java.lang.String, java.util.List, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for GateKeeperRuntimeCache.setGateKeeperValue$default (rex: <span class="sympad"><a href=".*com/facebook/internal/gatekeeper/GateKeeperRuntimeCache.html#setGateKeeperValue$default\(com.facebook.internal.gatekeeper.GateKeeperRuntimeCache, java.lang.String, java.lang.String, boolean, int, java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for InstrumentData.Type.valueOf (rex: <span class="sympad"><a href=".*com/facebook/internal/instrument/InstrumentData.Type.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AndroidRootResolver.ListenableArrayList.add (rex: <span class="sympad"><a href=".*com/facebook/internal/logging/dumpsys/AndroidRootResolver.ListenableArrayList.html#add\(java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AndroidRootResolver.ListenableArrayList.contains (rex: <span class="sympad"><a href=".*com/facebook/internal/logging/dumpsys/AndroidRootResolver.ListenableArrayList.html#contains\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AndroidRootResolver.ListenableArrayList.contains (rex: <span class="sympad"><a href=".*com/facebook/internal/logging/dumpsys/AndroidRootResolver.ListenableArrayList.html#contains\(java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AndroidRootResolver.ListenableArrayList.indexOf (rex: <span class="sympad"><a href=".*com/facebook/internal/logging/dumpsys/AndroidRootResolver.ListenableArrayList.html#indexOf\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AndroidRootResolver.ListenableArrayList.indexOf (rex: <span class="sympad"><a href=".*com/facebook/internal/logging/dumpsys/AndroidRootResolver.ListenableArrayList.html#indexOf\(java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AndroidRootResolver.ListenableArrayList.lastIndexOf (rex: <span class="sympad"><a href=".*com/facebook/internal/logging/dumpsys/AndroidRootResolver.ListenableArrayList.html#lastIndexOf\(android.view.View\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AndroidRootResolver.ListenableArrayList.lastIndexOf (rex: <span class="sympad"><a href=".*com/facebook/internal/logging/dumpsys/AndroidRootResolver.ListenableArrayList.html#lastIndexOf\(java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AndroidRootResolver.ListenableArrayList.remove (rex: <span class="sympad"><a href=".*com/facebook/internal/logging/dumpsys/AndroidRootResolver.ListenableArrayList.html#remove\(int\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AndroidRootResolver.ListenableArrayList.remove (rex: <span class="sympad"><a href=".*com/facebook/internal/logging/dumpsys/AndroidRootResolver.ListenableArrayList.html#remove\(int\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AndroidRootResolver.ListenableArrayList.remove (rex: <span class="sympad"><a href=".*com/facebook/internal/logging/dumpsys/AndroidRootResolver.ListenableArrayList.html#remove\(java.lang.Object\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for DefaultAudience.valueOf (rex: <span class="sympad"><a href=".*com/facebook/login/DefaultAudience.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for LoginTargetApp.valueOf (rex: <span class="sympad"><a href=".*com/facebook/login/LoginTargetApp.html#valueOf\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for FacebookSdk.GraphRequestCreator.createPostRequest (rex: <span class="sympad"><a href=".*com/facebook/FacebookSdk.GraphRequestCreator.html#createPostRequest\(com.facebook.AccessToken, java.lang.String, org.json.JSONObject, com.facebook.GraphRequest.Callback\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for CodelessManager.CodelessSessionChecker.checkCodelessSession (rex: <span class="sympad"><a href=".*com/facebook/appevents/codeless/CodelessManager.CodelessSessionChecker.html#checkCodelessSession\(java.lang.String\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for AppLinkResolver.getAppLinkFromUrlInBackground (rex: <span class="sympad"><a href=".*com/facebook/bolts/AppLinkResolver.html#getAppLinkFromUrlInBackground\(android.net.Uri\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no match for Continuation.then (rex: <span class="sympad"><a href=".*com/facebook/bolts/Continuation.html#then\(com.facebook.bolts.Task<TTaskResult>\)".*\(([^\(\)]*)\))
1>CLASSPARSE : warning : no document found : ..\..\externals\facebook-core-docs\com/facebook/ppml/receiver/IReceiverService.html
```

Reply:
To speed up discussion, feel free to contact me on https://gitter.im